### PR TITLE
[MCP23017] Allow choosing I2C bus numbers other than 0 and 1

### PR DESCRIPTION
**Pull request by Jockra**
[MCP23017] Allow choosing I2C bus numbers other than 0 and 1

If additional I2C busses have to be used because of a limitation of 8 MCP23017 on the same bus (8 available addresses) those additional bus numbers should be chooseable. It is only nescessary to add further options in the GUI

### CODE REVIEW

**Comment by Jockra** [comment]
I updated now the readme and extended to options to be able to choose I2c6. According the Raspberry Pi overlay documentation is this possible.

File: bundles/org.openhab.binding.mcp23017/README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ This binding allows you to have native access for MCP23017 I/O expander on I2C b
 It was tested with Raspberry Pi 2 and Raspberry Pi 3, but probably should work with other devices supported by [Pi4J](https://pi4j.com/) library.
 
 On Raspberry Pi the user on which openHAB is running (default user name is "openhab") needs to be added to groups "i2c" and  "gpio".
+As the MCP23017 has 3 address pins, you are restricted to 8 devices on a I2C bus.
+To use more devices you have to open further I2C busses. Therefore you can use overlays to enable bit banging I2C busses on the Raspberry Pi connector, up to I2C6.
```
Comment (J-N-K) [inline bundles/org.openhab.binding.mcp23017/README.md]: Please use a new line for each sentence. I also don't understand what excatly is needed to enable the overlays. Is this just a configuration option the user has to set?

Comment (Jockra) [reply bundles/org.openhab.binding.mcp23017/README.md]: To be more precise: it is a Linux kernel device tree overlay which can be appended in the /boot/config.txt file of RFS. It is also a configuration of alternative pin functions. I´ve added therefore  the link to available overlays

Comment (Jockra) [reply bundles/org.openhab.binding.mcp23017/README.md]: In my case I ´ve added  
dtoverlay=i2c-gpio,i2c_gpio_sda=20,i2c_gpio_scl=21,i2c_gpio_delay_us=1
to config.txt s. [examples](https://www.instructables.com/id/Raspberry-PI-Multiple-I2c-Devices/)

Comment (J-N-K) [reply bundles/org.openhab.binding.mcp23017/README.md]: Thanks. That makes it more clear

Comment (J-N-K) [inline bundles/org.openhab.binding.mcp23017/README.md]:
```suggestion
To use more devices you have to open further I2C busses.
Therefore you can use overlays to enable bit banging I2C busses on the Raspberry Pi connector, up to I2C6.
```

**Comment by J-N-K** [comment]
Travis fails unrelated in feed tests

**Pull request merged**